examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge/CastingServerBridge.mm

```diff
@@ -2083,7 +2137,7 @@ - (void)applicationBasic_readVendorID:(ContentApp * _Nonnull)contentApp
     [_readSuccessCallbacks setObject:successCallback forKey:@"applicationBasic_readVendorID"];
```

Comment:
Would it be cleaner if we just entirely skipped storing these callbacks in the _readSuccess/failureCallbacks property?